web/src/RootSwitcher.tsx

```diff
@@ -86,6 +87,7 @@ const RootSwitcher = ({ setContentLanguage }: RootSwitcherProps): ReactElement =
 
   return (
     <Suspense fallback={<LoadingSpinner />}>
+      <JsonLdWebSite />
```

Comment:
I would rather put it in the Helmet.tsx file, don't think RootSwitcher is the best place for that (at least I would not expect it here).

Reply:
Okay, makes sense. I think `App` would be the better place then. Please also add a comment with this reasoning why it has to stay there.